libcodechecker/analyze/statistics_collector.py

```diff
@@ -97,13 +93,18 @@ class SpecialReturnValueCollector(object):
     # Checker name which runs the analysis.
     checker_analyze = 'statisticsbased.SpecialReturnValue'
 
-    def __init__(self):
+    def __init__(self, stats_min_sample_count=10,
+                 stats_relevance_threshold=0.85):
```

Comment:
These default values are already provided at the argument parser at the outermost level. These are redundant.